tools/psa/release.py

```diff
@@ -74,7 +74,12 @@ def _get_target_info(target):
     if not os.path.exists(delivery_dir):
         raise Exception("{} does not have delivery_dir".format(target))
 
-    return tuple([TARGET_MAP[target].name,
+    if toolchain:
```

Comment:
Add assertion that the target actually supports the toolchain

Reply:
done